src/index.ts

```diff
@@ -91,14 +96,21 @@ export function doubleCsrf({
   const generateToken: CsrfTokenCreator = (
     req: Request,
     res: Response,
-    { overwrite = false, validateOnReuse = true } = {},
+    {
+      cookieOptions = defaultCookieOptions,
+      overwrite = false,
+      validateOnReuse = true,
+    } = {},
   ) => {
     const { csrfToken, csrfTokenHash } = generateTokenAndHash(req, {
       overwrite,
       validateOnReuse,
     });
     const cookieContent = `${csrfToken}|${csrfTokenHash}`;
-    res.cookie(cookieName, cookieContent, { ...cookieOptions, httpOnly: true });
+    res.cookie(cookieName, cookieContent, {
```

Comment:
here defaultCookieOptions should go before cookieOptions right?

Reply:
Yes. Yes they should. 😂